ide-migration/server/migration/api/migration-service.mjs

```diff
@@ -108,7 +108,17 @@ export class MigrationService {
     }
 
     _buildHDIContainerName(duName, projectName) {
-        return `${duName}_${projectName}`.toUpperCase();
+        const groupAndContainerNamePattern = /[^\w_]/g;
```

Comment:
Wouldn't `/[^\w]/g` be enough here? `\w` matches the `_` too

Reply:
It is. Fixed